assume/world.py

```diff
@@ -383,6 +383,13 @@ async def async_add_unit(
                 ):
                     self.learning_role.rl_strats[id] = bidding_strategies[product_type]
 
+                    # if we have learning strategy we need to assign the powerplant to one  unit_operator handling all leanring units
+                    self.logger.warning(
+                        f"Your chosen unit-operator {unit_operator_id} for the learning unit {id} was overwritten, with Operator RL, since al learning units need to be handeled by one unit operator."
```

Comment:
```suggestion
                        f"Your chosen unit-operator {unit_operator_id} for the learning unit {id} was overwritten, with Operator RL, since all learning units need to be handeled by one unit operator."
```

Reply:
Do we need this warning, if the operator is overwritten, even if "Operator RL" was chosen already?

Reply:
Good point - we only need it `if unit_operator_id != "Operator RL"`

While we are at it - I don't like spaces in names a lot and would prefer "Operator-RL" instead if that is fine for you?